bpm_projects_api/core/security.py

```diff
@@ -0,0 +1,93 @@
+from datetime import datetime, timedelta
+from functools import wraps
+
+import jwt
+from flask import request, current_app as app, make_response, Blueprint
+from flask.json import jsonify
+from flask_restplus import abort
+from jwt import DecodeError, ExpiredSignatureError
+
+ns = Blueprint('security', __name__)
+
+authorizations = {
```

Comment:
Where this will be used?

Reply:
Yes, its used in '/login' and it returns the *token*

Reply:
That part actually wont be used that way unless it be for testing purposes. Each app will communicate with an auth endpoint to get a token. This microservice will communicate with that endpoint as well when the secret key (used to encrypt the token) expires.